docs/chapters/baseplatform/building-PELUX-sources.rst

```diff
@@ -54,4 +58,84 @@ Intel NUC are ``.wic`` images, which are complete disk-images. For the Raspberry
 Pi 3, the preferred format is ``.rpi-sdimg`` which can be directly written to
 the SD card.
 
+Building with Vagrant
+---------------------
+
+When we build internally at Pelagicore in our CI system, we use vagrant.  Please
+note that we only run this setup in a GNU/Linux system at Pelagicore. It should
+still work under Windows or OSX, but we haven't tried it.
+
+Dependencies:
+^^^^^^^^^^^^^
+
+* Vagrant
+* Docker
+* Virtualization enabled in BIOS
+
+Procedure:
+^^^^^^^^^^
+
+1. Clone the pelux-manifests git repository.
+2. Start vagrant
+
+.. code-block:: bash
+
+    vagrant up
+
+
+3. Set variables to be used below
+
+.. code-block:: bash
+
+    export bitbake_image="core-image-pelux-minimal"
+    export yoctoDir="/home/vagrant/pelux_yocto"
+    export manifest="pelux.xml"
```

Comment:
Now with the variant change it seems that instead of having the manifest variable I would prefere the variant as a variable, or both.

Reply:
Probably a good idea. I'll make some changes (noticed that the readme was built on old scripts also, so will update other things).